drivers/mm/mm_drv_intel_adsp_mtl_tlb.c

```diff
@@ -642,6 +642,13 @@ static int sys_mm_drv_mm_init(const struct device *dev)
 
 	ARG_UNUSED(dev);
 
+	/*
+	 * Change size of avalible physical memory acording to fw register information
+	 * in runtime.
+	 */
+	uint32_t avalible_memory_size = mtl_hpsram_get_bank_count() * SRAM_BANK_SIZE;
```

Comment:
I am sure this does not exist in upstream zephyr, try `ace_hpsram_get_bank_count`

Reply:
Was working with bad revision - fixed.